File: python/fizzbuzz.py
for i in range(0, 100):
    output = ""

    if i % 3 == 0:
        output += "Fizz"
    if i % 5 == 0:
        output += "Buzz"

    if output == "":
        print(i)
    else:
        print(output)

File: javascript/fizzbuzz.js
for (i = 0; i <= 100; i++) {
  const str = `${i % 3 === 0 ? 'Fizz' : ''}${i % 5 === 0 ? 'Buzz' : ''}`;

  if (str) {
    console.log(str);
  } else {
    console.log(i);
  }
}
